runtime/vm/JVMImage.hpp

```diff
@@ -94,6 +95,9 @@ class JVMImage
 	ImageTableHeader* getClassLoaderTable(void) { return WSRP_GET(_jvmImageHeader->classLoaderTable, ImageTableHeader*); }
 	ImageTableHeader* getClassTable(void) { return WSRP_GET(_jvmImageHeader->classTable, ImageTableHeader*); }
 	ImageTableHeader* getClassPathEntryTable(void) { return WSRP_GET(_jvmImageHeader->classPathEntryTable, ImageTableHeader*); }
+
+	bool getIsWarmRun() const { return _isWarmRun; }
```

Comment:
Only reason I kept them, was because we had a boolean as a member in our class. And I wasn't sure if we were trying to avoid checks by keeping the boolean.
I will remove it